index.html

```diff
@@ -26,7 +26,26 @@
       <li>Capitol Hill</li>
       <li>Alki</li>
     </ul>
-    <p id="inspiration">Remember! Life might get hard, but always swim upstream!</p>
-    <script src="app.js" type="text/javascript"></script>
+    <div class="form">
+      <form id="new-store-form">
+        <fieldset>
+          <legend>Enter New Cookie Store Info</legend>
+            <label for="newStoreName">Cookie Store Name:</label>
```

Comment:
Technically, the for attribute needs to take the id of the form element it labels, not the name of it.